docs/dashboards/frequently-asked-questions.md

```diff
@@ -0,0 +1,108 @@
+FAQ
+===
+### How to connect store to the other components
+First, you need to create the correct type of store using the right constructor, and pass the data reference. More about this topic in the DataStore
```

Comment:
I would add a reference to the `DataStore` docs and give a simple snippet of how this should look like.